.travis.yml

```diff
@@ -90,7 +90,7 @@ jobs:
     # Run benchmarks and publish the result to github pages.
     - name: "benchmarks"
       script:
-        - cargo build --release --locked --all-targets
+        - cargo build -vv --release --locked --all-targets
```

Comment:
Can you leave this out? It ends up being a ton of 99% of the time useless stuff to scroll thru

Reply:
I can try it again, but last time I've tried travis timed out because it didn't receive a build output for 10 minutes and building `v8` from scratch takes unfortunately that long (because it can't use the old cache here).

Reply:
That’s a good reason to keep it!